src/components/navbar.tsx

```diff
@@ -0,0 +1,53 @@
+import SearchIcon from "./svgs/search";
```

Comment:
merge main into this branch, then get the icons from the /components/icons folder